index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
Ah. Hmm, I'd make the bytecode regex less strict, at least for now. Effectively because link references can contain any character in between those underscores, at the moment. You could also write a regex that takes into account the underscores and the ability to add any character between those underscores (and that the reference length should be 40 characters total), but that sounds like too much work for a single regex. I'm fine with either option at this point.